app/test/unit/git/status-test.ts

```diff
@@ -86,25 +86,30 @@ describe('git/status', () => {
       expect(files[1].path).to.equal('docs/OVERVIEW.md')
     })
 
-    it('Handles at least 10k untracked files without failing', async () => {
-      const numFiles = 10000
-      const basePath = repository!.path
-
-      await mkdir(basePath)
-
-      // create a lot of files
-      const promises = []
-      for (let i = 0; i < numFiles; i++) {
-        promises.push(
-          FSE.writeFile(path.join(basePath, `test-file-${i}`), 'Hey there\n')
-        )
-      }
-      await Promise.all(promises)
-
-      const status = await getStatusOrThrow(repository!)
-      const files = status.workingDirectory.files
-      expect(files.length).to.equal(numFiles)
-    })
+    it(
+      'Handles at least 10k untracked files without failing',
+      async () => {
+        const numFiles = 10000
+        const basePath = repository!.path
+
+        await mkdir(basePath)
+
+        // create a lot of files
+        const promises = []
+        for (let i = 0; i < numFiles; i++) {
+          promises.push(
+            FSE.writeFile(path.join(basePath, `test-file-${i}`), 'Hey there\n')
+          )
+        }
+        await Promise.all(promises)
+
+        const status = await getStatusOrThrow(repository!)
+        const files = status.workingDirectory.files
+        expect(files.length).to.equal(numFiles)
+      },
+      // needs a little extra time on CI
+      25000
```

Comment:
the only non-formatting change in this file

Reply:
https://github.com/prettier/prettier/issues/5061